### ISSUE TITLE
Implement HmIPW-DRS4

### ISSUE THREAD

**Issue by I3astel** [hahn-th/homematicip-rest-api#436]
Please implement Homematic IP Wired Switch Actuator - 4 channels: HmIPW-DRS 4

[hmip-config_YYYY.zip](https://github.com/coreGreenberet/homematicip-rest-api/files/8120302/hmip-config_YYYY.zip)

**Comment by sschiffer** [comment]
I'd be interested in this device, too. 

In order to use all HmIPW devices also in Apple Home, I installed Homebridge incl. the [Homematic IP plugin](https://github.com/marcsowen/homebridge-homematicip). Unfortunately, this plugin does not yet provide full compatibility for all HmIP Wired devices that are already covered by this library.

But in order to support progress there, I found out the HmIPW-DRS4 would also have to be added here, first.

I'll attach my config here, soon. Maybe, I'm also able to support in terms of development...
